tasks/manage.yml

```diff
@@ -50,6 +50,30 @@
   set_fact:
     dsa_control_argument: "-m 'AntiMalwareManualScan:true'"
   when: action == 'scan-for-malware'
+  
+# *********************************************************************
+# * Set a policy by name
+# *********************************************************************
+- name: Set policy by name
+  set_fact:
+    dsa_control_argument: "-m 'policy:{{ policy }}'"
```

Comment:
You may want to use "policy_name" instead of "policy" to make the purpose of this parameter more clear.